lib/middlewares/haspermission.middleware.js

```diff
@@ -0,0 +1,15 @@
+const permissionValidation = require('../services/permissionvalidation.service');
+
+function hasPermission(requiredPermissions, source) {
+  return async (req, res, next) => {
+    try {
+      await permissionValidation(req.get('authorization'), requiredPermissions, source);
```

Comment:
The token will not always be passed in the Authorization header. For example, a BFF often stores the user information (including the token) in the session, and API calls to the BFF only contain a cookie.

So it should be configurable where to get the token.

Reply:
Configureerbaar gemaakt

Reply:
In de Authorization header zal het token altijd prefix "Bearer " hebben. Maar als het token in de sessie wordt bijgehouden, zal dat meestal zonder "Bearer " zijn. Dus daar best nog een oplossing voor voorzien.